src/rabbit_msg_store_ets_index.erl

```diff
@@ -68,8 +68,8 @@ delete_object(Obj, State) ->
     true = ets:delete_object(State #state.table, Obj),
     ok.
 
-delete_by_file(File, State) ->
-    MatchHead = #msg_location { file = File, _ = '_' },
+cleanup_undefined_file(State) ->
```

Comment:
The function itself is an implementation detail. And I would like to point it out. The better name would be `clean_up_temporary_reference_count_entries_without_file`